frontend/src/composables/useShootDns.js

```diff
@@ -45,6 +45,21 @@ export const useShootDns = (manifest, options) => {
     deleteExtension,
   } = useShootExtensions(manifest)
 
+  const setDnsServiceExtension = providers => {
+    const providerConfig = {
+      apiVersion: 'service.dns.extensions.gardener.cloud/v1alpha1',
+      kind: 'DNSConfig',
+      syncProvidersFromShootSpecDNS: false,
+    }
+    if (providers) {
+      providerConfig.providers = providers
+    }
+    setExtension({
+      type: 'shoot-dns-service',
+      providerConfig,
+    })
+  }
```

Comment:
```suggestion
  function setDnsServiceExtensionProviders (providers) {
    setExtension({
      type: 'shoot-dns-service',
      providerConfig: {
        apiVersion: 'service.dns.extensions.gardener.cloud/v1alpha1',
        kind: 'DNSConfig',
        syncProvidersFromShootSpecDNS: false,
        providers,
      },
    })
  }
```
It make no difference if the property does not exist or the value is `undefined`